src/main/java/it/pagopa/interop/probing/scheduler/producer/ServicesSend.java

```diff
@@ -29,7 +29,6 @@ public void sendMessage(PollingActiveEserviceResponse service) throws IOExceptio
     SendMessageRequest sendMessageRequest = new SendMessageRequest().withQueueUrl(sqsUrl)
         .withMessageBody(objectMapper.writeValueAsString(service));
     amazonSQS.sendMessage(sendMessageRequest);
-    log.info("Service with id" + service.getId() + " with version " + service.getVersionNumber()
-        + " has been published in SQS.");
+    log.info("Service with id" + service.getId() + " has been published in SQS.");
```

Comment:
just a note: from the message it seems that the logged id is the `eserviceId`, while it is actually the id of the record in the table.
This difference should be clear in the message